recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,105 @@
+arm_variant_type: # [aarch64]
+  - sbsa          # [aarch64]
+
+# The added keys below are needed to enable ppc64le/aarch64 CUDA builds; the linux64
+# and win64 builds are already handled in the conda-forge-pinning feedstock. Once we
+# migrate to enable CUDA builds on all supported platforms, these can be removed too.
+
+docker_image:                                           # [os.environ.get("BUILD_PLATFORM", "").startswith("linux")]
```

Comment:
Currently, in the recipe, it is specified that building is skipped for CUDA Toolkit versions other than 11.2
```
skip: True   # [cuda_compiler_version != "11.2"]
```
So I think we don't need that many keys here?

Reply:
Yes I'll clean it up later.

Reply:
Done in 51a57ec.